docs/running.md

```diff
@@ -29,9 +29,17 @@ mkdir test-network
 ### Create a new Genesis file
 
 First, create a new `genesis.json` file devoid of accounts and validators. The `--base-fee` here is completely arbitrary.
+The `--power-scale` value of `0` means we'll grant 1 voting power per 1 FIL; to use more precision, we can set it to `3`
```

Comment:
To clarify, in the contract I ended going with `power_scale` as the accuracy for the power table so as you mention `0 == FIL, 3 == miliFil, 12 == atto`.

Reply:
atto would be 18, but otherwise sounds like you're using it the same way.

Reply:
Already fixed in the contract, I keep saying that atto is 12 because I am stupid :)